docs/sources/upgrading/_index.md

```diff
@@ -33,6 +33,14 @@ The output is incredibly verbose as it shows the entire internal config struct u
 
 ## Main / Unreleased
 
+### Loki
+
+#### Shutdown marker file
+
+A shutdown marker file can be written by the `/ingester/prepare_shutdown` endpoint.
+If the new `ingester.shutdown_marker_path` config setting has a value that value is used.
```

Comment:
```suggestion
If the new `ingester.shutdown-marker-path` config setting has a value that value is used.
```

Reply:
the config setting is `shutdown_marker_path`, `shutdown-marker-path` is its flag counterpart